### ISSUE TITLE
Server/Client crash together

### ISSUE THREAD

**Issue by joshbgosh10592** [OpenRCT2/OpenRCT2#10226]
**OS:** [Server is Ubuntu, client is macOS]
**Version:** [Server: /Client: 0.2.4-59]
**Commit/Build:** [Client: 9df66b1]

Server/client crash so hard that they both end up crashing. The most recent time, my client was the only user connected, and I was enabling cheat (disable vandalism, rides breaking down).
This is reproducible both in multiplayer and single. 

- [ ] Reproducible in RCT2 (vanilla)?
- [ ] Specific to multiplayer?

**Steps to reproduce:**
1. Open attached save file
2. Sit and wait for about 2 minutes

**Dump file**
Not sure where to get the dump file on either OS. I don't see any dmp files where config.ini is located. I can provide if you tell me where it is.

**Save game:**
[Woodland_Thrills_crash.sv6.txt](https://github.com/OpenRCT2/OpenRCT2/files/3839069/Woodland_Thrills_crash.sv6.txt)

**Comment by janisozaur** [comment]
Ubuntu is by default using systemd and you should be able to retrieve the coredump using: `coredumpctl -o <output file> dump <path to openrct2 binary that crashed>`

**Comment by joshbgosh10592** [comment]
How would I know which openrct2 binary crashed?
Also, looks like coredumpctl isn't actually installed by default. Should I install it and grab the dump (of which I'll need you to tell me how to determine the binary in question)?

**Comment by joshbgosh10592** [comment]
@janisozaur I'd like to provide more information to get this resolved, but I don't know where to get that information. But the save file is transferable (I'm able to transfer that to my mac in single player mode and able to get it to crash as well.)

**Comment by janisozaur** [comment]
Ok, just by running that save? Headless?

**Comment by joshbgosh10592** [comment]
Yup, I was able to reproduce it just by running the save, both headless (Ubuntu) or with a GUI (macOS).
